pkg/cmd/variable/create/create.go

```diff
@@ -0,0 +1,219 @@
+package create
```

Comment:
Variables moved away from set as `set` is synonymous with `PUT` operation(secret create/update both are handled via `PUT`), whereas variable create and update are `POST` and `PATCH` respectively.

Reply:
As we are abstracting away the concepts of `PUT`/`POST`/`PATCH` from the end user I don't think we need to worry about that. They do not necessarily care what type of HTTP request we are making under the covers. I think it is much better to be consistent with our other commands so that the end user experience seems polished and uniform. Additionally, it seems that the current `create` and `update` variable commands offer almost the exact same interface for our users and I think it makes sense to combine them for simplicity of use. The user won't need to think about using `create` or `update` they can call `set` and it will do the right things, either creating a new variable or updating an existing one.